tests/infra_test.go

```diff
@@ -57,6 +57,7 @@ var _ = Describe("Infrastructure", func() {
 		var vmi *v1.VirtualMachineInstance
 		var pod *k8sv1.Pod
 		var metricsURL string
+		var cachedMetrics string
```

Comment:
I'm not sure about caching. It goes against the basic tenet to exercise the flows as much as we can, and adds another shared resource. What's the benefit of using cache? How much time do we save?

Reply:
Yeah, good point, I'm not sure about it myself :) it surely saves time but at the expense of what you have mentioned. I'll need to measure the time saved

Reply:
Removed the caching